data_frame_reshape.Rmd

```diff
@@ -0,0 +1,112 @@
+
```

Comment:
- The file should not contain a YAML header nor a --- line.
- The first line should start with a single hashtag #, followed by a single whitespace, and then the title.
- The second line should be blank, followed by the author name(s).
- If the file contains a setup chunk in .Rmd file, it should not contain a setup label. (The bookdown render will fail if there are duplicate chunk labels.)
i.e. use {r, include=FALSE} instead of {r setup, include=FALSE}.

Above are some of the issues in this PR. Please, thoroughly examine the submission instructions and resubmit. https://jtr13.github.io/cc21fall2/github-submission-instructions.html.

Reply:
@JingwenB

Reply:
Sorry I missed that， I'll resubmit soon